server/app/controllers/FeatureFlagOverrideController.java

```diff
@@ -0,0 +1,32 @@
+package controllers;
+
+import annotations.FeatureFlagOverrides;
+import auth.Authorizers;
+import javax.inject.Inject;
+import org.pac4j.play.java.Secure;
+import play.mvc.Http.Request;
+import play.mvc.Result;
+
+/**
+ * Allows for overriding of feature flags by the CiviForm Admin via HTTP request.
+ */
+public class FeatureFlagOverrideController extends CiviFormController {
+  private final FeatureFlagOverrides overrides;
+
+  @Inject
+  public FeatureFlagOverrideController(FeatureFlagOverrides overrides) {
+    this.overrides = overrides;
+  }
+
+  @Secure(authorizers = Authorizers.Labels.CIVIFORM_ADMIN)
+  public Result enable(Request request, String flagName) {
+    overrides.setOverride(flagName, "true");
```

Comment:
Right now, the only use case for overrides is to always use a boolean. Until we have a use case otherwise, would it make sense for setOverride to take a string key and a boolean value?

Reply:
I was tacitly trying to follow the configuration class which I think just does bare strings, and relies on the user doing the right thing wrt to the value in the configuration file and code getting with the right type...

I don't have a strong opinion either way though if you think overloads in that class are better.